sdk/communication/tests.yml

```diff
@@ -10,9 +10,13 @@ jobs:
       - name: azure-communication-chat
         groupId: com.azure
         safeName: azurecommunicationchat
+      - name: azure-communication-sms
+        groupId: com.azure
+        safeName: azurecommunicationsms
     ServiceDirectory: communication
     EnvVars:
       AZURE_TEST_MODE: LIVE
       SKIP_PHONENUMBER_INTEGRATION_TESTS: TRUE
       SKIP_LIVE_TEST: TRUE
-
+      COMMUNICATION_CONNECTION_STRING: $(python-communication-connection-string)
```

Comment:
python? Didn't we decide to have a separate one for all languages? and if we want to use the common then can we update the name to reflect something common, like communication-tests-connection-string